REQUIRE

```diff
@@ -1,3 +1,4 @@
-julia 0.7
+julia 0.6
```

Comment:
Great with the bugs and tests :-), but do we really need 0.6? and Compat :anguished: ?

Reply:
I do - until Claire has everything converted to 1.0, all of the Diversity updates need to work back to 0.6. Hopefully it won’t be for long, but several packages she relies on (e.g. JuliaDB) aren’t there yet.